lib/options.go

```diff
@@ -353,3 +354,18 @@ func (o Options) Apply(opts Options) Options {
 	}
 	return o
 }
+
+func NullDecoder(f reflect.Type, t reflect.Type, data interface{}) (interface{}, error) {
```

Comment:
That's great, I'm very happy that `mapstructure` allows this! 

Can you move this in the subfolder [here](https://github.com/loadimpact/k6/tree/master/lib/types) and add support for the `NullDuration` type that's defined there. It's not used in the influxdb config, but a lot of other null-able configs use it. And also a unit test or two? I'm not familiar with how mapstructure works, seems [somewhat complex](https://godoc.org/github.com/mitchellh/mapstructure#DecoderConfig), but I'm not sure this would correctly handle cases where the actual supplied data is of a different type than the receiver type. You're only checking the destination type, not the source/data type and you're directly casting the values without checking if that's successful.